product-service/handlers/get-product-by-id.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { getProductById } from './get-product-by-id';
+import { products } from '../products';
+
+describe('getProductById', () => {
+  describe('if product exists', () => {
+    it('should return response with code 200 and correct product', async () => {
+      const fakeEvent = {
+        pathParameters: { productId: `${products[0].id}` }
+      };
+      const result = await getProductById(fakeEvent as any, null, null);
+
+      expect(result).toEqual({
+        statusCode: 200,
```

Comment:
You may consider using some npm package with predefined status codes constants.

Reply:
Thanks, great idea 👍